.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: enhancement
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
```

Comment:
Imo it would be better to talk about use cases instead of problems.

Also, [ex vs eg](https://blog.prepscholar.com/ie-vs-eg-vs-ex-definition#:~:text=What%20Does%20Ex.-,Mean%3F,is%20used%20to%20introduce%20exercises.)

```suggestion
**Is your feature request related to a use case? Please describe.**
A clear and concise description of what the use case is. E.g. I'd want to enable my users to [...]
```